tests/README.md

```diff
@@ -1,12 +1,44 @@
-# Basic opendatahub tests and how to use them
+# Running containerized tests
 
-This test repository is meant to be used with the
-test runner [located here](https://github.com/tmckayus/peak),
-see the README.md there for complete information.
+Running the tests this way assumes that you have an active kubeadmin login
+on the cluster that you want to run the tests against and that you have podman
+installed.  (If you prefer docker, you can edit the Makefile to replace podman
+with docker).
 
-The intial tests here verify deployment of a KFDef instance
+Run the following:
 
-# Quick start
+```
+cd tests
+make build
+make run
+```
+
+To cleanup the installation after a test run, you can run:
+```
+make clean
+```
+
+## Customizing test behavior
+
+Without changes, the test image will run `$HOME/peak/installandtest.sh` which
+handles setting up the opendatahub-operator and then creating the KfDef found in
+tests/setup/kfctl_openshift.yaml.  If you want to modify your test run, you
+might want to change those files to get the behavior that you're looking for.
+After you make changes, you will need to rebuild the test image with `make build`.
+
+If you'd like to run the tests against an instance that already has Open Data Hub installed,
+you set `SKIP_INSTALL=true` and that will cause the test run
+to skip the installation process and will only run the tests.  example: `make run SKIP_INSTALL=true`
+
+If you'd like to run a single test instead of all tests, you can
+set the TESTS_REGEX variable `TESTS_REGEX=<name of the test to run>`.  That will
+only run the test that you specify instead of all of the tests.  example: `make run TESTS_REGEX=grafana`
```

Comment:
Can you add a note for people to take a look into `Makefile` at the other variables they can configure? There is more now like `ODHPROJECT`, OpenShift user credentials etc.